Automattic/jetpack
# Repo Gardening support ref task: update label name.

## Proposed changes:

Let's update the name used to denote issues that include customer references.
### Other information:


## Jetpack product discussion

Internal reference: p1691543440826329-slack-CQD1HH4MA

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Not much to test until this PR gets merged.
* Before we merge it though, we should update the label name in the repos that use this task so as to not create a new label while losing all past references.

Repo references:
- [ ] Jetpack
- [ ] wp-calypso
- [ ] themes
- [ ] crowdsignal
- [ ] sensei
- [ ] WP-Job-Manager
